config/routes.rb

```diff
@@ -16,6 +19,10 @@
   get 'documents/catalog', to: 'catalog#index'
   get 'documents/catalog/image/:eid', to: 'catalog#image'
   get 'documents/catalog/reference/:eid', to: 'catalog#reference'
+  post 'anthology_add', to: 'documents#anthology_add'
+  post 'user_anthology_add', to: 'users#anthology_add'
+  post 'remove_user', to: 'users#remove_user'
+  post 'remove_doc', to: 'anthologies#remove_doc'
 
```

Comment:
Routing and controller semantics are a little messy here, but let's go with it and clean up later if/as necessary